core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -34,7 +35,8 @@ angular.module('oppia').directive('imageUploader', [
         height: '@',
         onFileChanged: '=',
         errorMessage: '@',
-        width: '@'
+        width: '@',
+        allowedImageType: '='
```

Comment:
We don't require two-way binding here, right? Maybe use '@' or '&'.

Reply:
Note: This comment has not been addressed.

Reply:
I am sorry for not noticing this. Done.

Reply:
@kevintab95  I am sorry for my mistake but I am reverting this to = because the argument is a list that is passed and the scope type should be = and not & or @.

Reply:
That's not correct. If the argument only needs to be read one-way, then '&' or '@' is more appropriate (and definitely possible). You might need to experiment a bit / look at other examples in the codebase to figure out how exactly to do it, though.

Reply:
I am sorry again I read that using = was the only way in a stackoverflow page and came to a conclusion. Now I used @ and JSON parse function to get the data. Done.